imports/plugins/included/product-variant/client/templates/products/products.js

```diff
@@ -172,3 +173,37 @@ Template.products.events({
     loadMoreProducts();
   }
 });
+
+Template.socialTimeline.onCreated(function () {
+  this.state = new ReactiveDict();
+  this.state.setDefault({
+    timeline: {}
+  });
+
+  this.autorun(() => {
+    this.subscribe("Packages");
+    const socialConfig = Packages.findOne({
+      name: "reaction-social"
+    });
+    this.state.set("timeline", socialConfig.settings.public.apps);
```

Comment:
What happens if socialConfig is 'undefined', you might want to put that into consideration.